test/functional/testlib.bash

```diff
@@ -4,10 +4,18 @@ FUNC_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 TEST_FILENAME=$(basename "$BATS_TEST_FILENAME")
 TEST_NAME=${TEST_FILENAME%.bats}
 THEME_DIRNAME="$BATS_TEST_DIRNAME"
+CACERT_DIR="$FUNC_DIR/test_certificates" # trusted key store path
+CLIENT_CERT_DIR="$TEST_NAME/target-dir/etc/swupd"
+CLIENT_CERT="$CLIENT_CERT_DIR/client.pem"
+
+PORT_FILE="$TEST_NAME/port_file.txt" # stores web server port
+SERVER_PID_FILE="$TEST_NAME/pid_file.txt" # stores web server pid
```

Comment:
same here, if these variables depend on a test to already exist they should go into the set_env_variables function, that way we can have all those variables together so they can be set up at once, once the test environment exists. The main idea behind this is to be able to use the library from a test script (which would be its main use) but also have the option of using it interactively from the command line if needed.

Reply:
I'll move these variables and the ones listed above, thanks